doc/how_to_guides/trac_ik/trac_ik_tutorial.rst

```diff
@@ -0,0 +1,48 @@
+TRAC-IK Kinematics Solver
+=========================
+
+`TRAC-IK <https://bitbucket.org/traclabs/trac_ik>`_ is an inverse kinematics solver developed by TRACLabs that combines two IK implementations via threading to achieve more reliable solutions than common available open source IK solvers.
+From their documentation:
+
+  (TRAC-IK) provides an alternative Inverse Kinematics solver to the popular inverse Jacobian methods in KDL.
+  Specifically, KDL's convergence algorithms are based on Newton's method, which does not work well in the presence of joint limits --- common for many robotic platforms.
+  TRAC-IK concurrently runs two IK implementations.
+  One is a simple extension to KDL's Newton-based convergence algorithm that detects and mitigates local minima due to joint limits by random jumps.
+  The second is an SQP (Sequential Quadratic Programming) nonlinear optimization approach which uses quasi-Newton methods that better handle joint limits.
+  By default, the IK search returns immediately when either of these algorithms converges to an answer.
+  Secondary constraints of distance and manipulability are also provided in order to receive back the "best" IK solution.
+
+The package `trac_ik_kinematics_plugin <https://bitbucket.org/traclabs/trac_ik/src/rolling-devel/trac_ik_kinematics_plugin/>`_ provides a ``KinematicsBase`` MoveIt interface that can replace the default KDL solver.
+Currently, mimic joints are *not* supported.
+
+Install
+-------
+
+The ``rolling-devel`` branch of the TRAC-IK repository has the latest ROS 2 implementation.
+For now, the repository must be built from source in your ROS 2 workspace, for example ``~/moveit2_ws``.
+
+```
+cd ~/moveit2_ws/src
+git clone -b rolling-devel https://bitbucket.org/traclabs/trac_ik.git
+```
+
+Usage
+-----
+
+- Find the MoveIt :doc:`kinematics.yaml </doc/examples/kinematics_configuration/kinematics_configuration_tutorial>` file created for your robot.
```

Comment:
same here, this doesn't render correctly to me.

Reply:
I built the HTML locally and this one shows up fine.

![image](https://github.com/ros-planning/moveit2_tutorials/assets/4603398/d9f13a95-bdcd-4087-99be-33e222178a56)